package.json

```diff
@@ -22,35 +24,30 @@
     "@types/nanoid": "^3.0.0",
     "@types/react": "^18.2.5",
     "@types/react-dom": "^18.2.3",
+    "@typescript-eslint/eslint-plugin": "^5.59.2",
+    "@typescript-eslint/parser": "^5.59.2",
     "canvas": "^2.11.2",
     "cypress": "^12.11.0",
+    "eslint": "^8.40.0",
     "eslint-config-prettier": "^8.8.0",
     "eslint-plugin-cypress": "^2.13.3",
     "eslint-plugin-prettier": "^4.2.1",
+    "eslint-plugin-react-hooks": "^4.6.0",
+    "eslint-plugin-react-refresh": "^0.4.1",
     "prettier": "^2.8.8",
-    "react-scripts": "^5.0.1",
     "typescript": "^5.0.4"
   },
   "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
+    "start": "vite",
+    "build": "vite build",
+    "preview": "vite preview",
     "test": "yarn run test:all",
-    "test:all": "yarn run test:type && yarn run test:lint && yarn run test:format && yarn run test:app --watchAll=false",
+    "test:all": "yarn run test:type && yarn run test:lint && yarn run test:format",
     "test:type": "tsc",
     "test:lint": "eslint --max-warnings=0 --ext .js,.ts,.tsx .",
     "test:format": "prettier 'src/*.(js|ts|tsx|css|scss)' --list-different",
-    "test:app": "react-scripts test --passWithNoTests",
```

Comment:
This should be replaced with "jest". Otherwise, we don't run tests, right?

Reply:
Yeah, you are right! Sorry about that 😅 
But since Jest requires a lot of configurations to work with React and TypeScript, and react-scripts handles that for us, I restored the script as it was (running with react-scripts)